README.md

```diff
@@ -29,6 +29,10 @@ A metaplugin such as [Multus](https://github.com/intel/multus-cni) gets the allo
 
 Accelerated Bridge plugin assumes that Linux Bridge is already exist and correctly configured on nodes.
 
+CNI plugin also supports VF bound to userspace driver (currently only vfio-pci) which may be utilized
+for virtualization use-case i.e kubevirt.
```

Comment:
nit: add link to https://github.com/kubevirt/kubevirt

Reply:
done